tests/integration/__fixtures__/hugo-site/package.json

```diff
@@ -6,6 +6,6 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "hugo-bin": "^0.101.0"
+    "hugo-bin": "^0.101.5"
```

Comment:
@danez I think you could also go with https://github.com/fenneclab/hugo-bin/releases/tag/v0.102.0 :)